File: Data Structures & Algorithms/sort-an-array/submission-0.java
class Solution {
    public int[] sortArray(int[] nums) {
         int minidx=0;
        for(int i=0;i<nums.length-1;i++)
        {
            minidx=i;
            for(int j=i+1;j<nums.length;j++)
            {
                if(nums[minidx]>nums[j])minidx=j;
            }
            if(minidx!=i)
            {
                int temp=nums[i];
                nums[i]=nums[minidx];
                nums[minidx]=temp;
            }
        }
        return nums;
    }
}

File: Data Structures & Algorithms/permutations/submission-0.py
class Solution:
    def permute(self, nums: List[int]) -> List[List[int]]:
        res=[]
        def solve(i):
            if i==len(nums):
                res.append(nums[:])
                return
            for j in range(i,len(nums)):
                nums[i],nums[j]=nums[j],nums[i]
                solve(i+1)
                nums[i],nums[j]=nums[j],nums[i]
        solve(0)
        return res
        